src/njs_vm.h

```diff
@@ -205,6 +205,7 @@ typedef struct {
 struct njs_vm_shared_s {
     njs_lvlhsh_t             keywords_hash;
     njs_lvlhsh_t             values_hash;
+    njs_flathsh_t            predefined_hash;
```

Comment:
Since predefined_hash is always constant, and is equal to `njs_predefined_hash` we do not need this field.
Because we already reference  `njs_predefined` directly. We can do the same for `njs_predefined_hash` too.